src/lib/class/Message/core.cpp

```diff
@@ -0,0 +1,82 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   core.cpp                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gle-mini <gle-mini@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/02/17 14:06:05 by mcourtoi          #+#    #+#             */
+/*   Updated: 2024/03/04 19:00:36 by gle-mini         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "Message.hpp"
+
+/**
+ * Destructor for the Message class.
+ */
+Message::~Message() {}
+
+/**
+ * Parses a single message from the given string.
+ *
+ * @param msg A constant reference to a std::string containing the message to be parsed.
+ */
+std::string const &Message::get_prefix() const { return _prefix; }
```

Comment:
Please, add the `this->` prefix everytime you access a field, or call a method from the class itself.